index.html

```diff
@@ -0,0 +1,144 @@
+            Insert a the quote valuta and rate to be added<br
+              class="zero-rate"
+            />
+          </p>
+
+          <span>
+            <label for="quote-name">Quote name</label>
+            <input
+              type="text"
+              id="quote-name"
+              step="0.001"
+              required
+              placeholder="quote-name"
+            />
+            <div></div>
+
+            <label for="quote-rate">Quote rate</label>
+            <input
+              type="number"
+              id="quote-rate"
+              step="0.001"
+              required
+              placeholder="quote-rate"
+            />
+            <div></div>
+          </span>
+          <br />
+
+          <div class="convert-currency_buttonc-section">
+            <button class="convert-currency_button" type="submit">
+              Add currency & rate
+            </button>
+          </div>
+        </fieldset>
+      </form>
+    </div>
+<BR></BR>
+
+    <!--valuta-converter part --------->
+    <div class="valuta-converter-based-on-rates">
+      <form
+        class="valuta-converter-based-on-rates_form"
+        id="valuta-converter-form"
+      >
+        <fieldset>
+          <legend>Money Conversion based on predefined exchange rate</legend>
+          <p>
+            <label>Money amount</label>
+            <input
+              id="money-amount"
+              type="number"
+              class="amount-field"
+              pattern=""
+              min="0"
+              step="0.001"
+              required
+              placeholder="Enter the amount 0.00"
+            />
+          </p>
+
+          <p></p>
+
+          <div>
+            <p class="explanations-in-forms">
+              Insert the quote valuta for conversion
+            </p>
+          </div>
+
+          <!--Base valuta-->
+          <label for="valuta-exchange-selction-base-valuta"
+            >The base valuta</label
+          >
+          <input
+            type="text"
+            id="base-currency"
+            value="EUR"
+            placeholder="EUR"
+            readonly
+          />
+
+          <div></div>
+
+          <!--Entering quote valuta -->
+          <label for="valuta-exchange-quote-valuta">Quote valuta</label>
+          <input
+            type="text"
+            id="valuta-exchange-quote-valuta"
+            step="0.001"
+            required
+            placeholder="Quote-valuta"
+          />
+          <div></div>
+          <br />
+
+          <!-- valuta convert button -->
+          <button class="valuta-converter_button" type="submit">
+            Convert the valuta
+          </button>
+          <br />
+          <br>
+          <!-- Display the converted amount -->
+          <span><label for="converted-amount">Converted amount: 
+          <p id="converted-amount-value"></p></label></span> 
+          
+        </fieldset>
+      </form>
+    </div>
+    <div></div>
```

Comment:
can be <div />

Reply:
I tried that, but it didn't accept that.

Reply:
https://github.com/katayouny/Currency-Converter/pull/1/files#r1426911027